gr-digital/python/digital/qam.py

```diff
@@ -19,7 +34,7 @@
 from .utils.gray_code import gray_code
 from .utils import mod_codes
 from . import modulation_utils
-from . import digital_python as digital
+from . import digital_swig as digital
```

Comment:
This is a 3.8 thing

Reply:
Ok, I was working off the 3.8 build code, so that's likely why